### ISSUE TITLE
Copy Paste in Firefox results in function getSelection() {     [native code] }

### ISSUE THREAD

**Issue by ckifer** [codesandbox/codesandbox-client#8231]
# 🐛 bug report

## Preflight Checklist

- [X] I have read the [Contributing Guidelines][contributing] for this project.
- [X] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [X] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

When I copy code in the beta editor my next paste does not contain my copied code, but rather

```js
function getSelection() {
    [native code]
}
```

## How has this issue affected you? What are you trying to accomplish?

Trying to copy code in beta sandbox and paste it elsewhere in the sandbox or outside of it 

### To Reproduce

1. Open any beta sandbox in Firefox
2. Copy code
3. Paste code anywhere
4. See bad result

### Link to sandbox: [link]() (optional)
[![Edit simple-area-chart (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/simple-area-chart-forked-qzfq4d)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |      latest beta editor     |
| Browser          |       latest Firefox     |
| Operating System |     latest OSX      |

Doesn't seem to be an issue in Chrome

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by JamesACS** [comment]
Hello @ckifer , 

I've just tried to reproduce this on Firefox on Mac in the new editor with both a Devbox and Sandbox and was unable to replicate the problem. The copy and paste facility works as expected. 

Is this still occurring for you?

**Comment by ckifer** [comment]
Hey @JamesACS 

Yep this is still happening. See video below, thanks!

https://github.com/codesandbox/codesandbox-client/assets/25180830/7889e5d2-aa84-4dee-a854-b0a227e1c806

**Comment by JamesACS** [comment]
Thanks @ckifer 

Is copy and pasting working normally on other websites?

**Comment by ckifer** [comment]
Yep, working as normal on other websites. Only when copying from codesandbox. Although I can do another test on another site that uses Monaco editor and see if it happens there too, will reply back later

**Comment by ckifer** [comment]
https://www.typescriptlang.org/play also has this issue so it isn't just codesandbox, but it seems specific to Monaco-esque editors

https://github.com/codesandbox/codesandbox-client/assets/25180830/26d85aa7-5b00-45b9-aa18-6999f3581742

**Comment by ckifer** [comment]
To add to this an error gets thrown on copy

`Uncaught Error: The operation is insecure.`

**Comment by kantum** [comment]
This is happening to me also recently, kind of frustrating...

**Comment by ckifer** [comment]
This was caused by an (internal) Firefox extension that I had installed, after disabling everything works okay. Still only an issue with codesandbox and not other sites

**Comment by JamesACS** [comment]
@ckifer please could you confirm the name of the extension? I'll add it to our report for this issue on our side

**Comment by ckifer** [comment]
@JamesACS no sorry I can't as it was an internal to my company extension that most should not have installed. Maybe @kantum could check if extensions are effecting him as well. I'd imagine any extension intercepting clipboard events might experience this.

**Comment by kantum** [comment]
For some reason I had `dom.event.clipboardevents.enabled` set to false in `about:config`.

Setting it back to default solved the problem.
